extensions/dependencies/guppy.html

```diff
@@ -8,3 +8,27 @@
     '/third_party/static/guppy-be6f14/src/transform.xsl',
     '/third_party/static/guppy-be6f14/src/symbols.json');
 </script>
+
+<style>
+    .guppy_active {
+      box-shadow: 0 0 5px rgba(81, 203, 238, 1) !important;
```

Comment:
Yes I checked that, the !important does override the JS. we don't need the box-shadow just that it gives a nice effect to the textbox and is kind of consistent with how the other inputs behave.